.changeset/little-wasps-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': minor
```

Comment:
Think we could move this to `patch` instead of `minor`?

Reply:
I wasn't sure which one to pick, since it changes ui behavior. Changed it to `patch`